sdks/python/apache_beam/testing/analyzers/perf_analysis.py

```diff
@@ -169,12 +182,11 @@ def run(config_file_path: Optional[str] = None) -> None:
 
   tests_config: Dict[str, Dict[str, Any]] = read_test_config(config_file_path)
 
-  big_query_metrics_fetcher = BigQueryMetricsFetcher()
 
-  for test_name, params in tests_config.items():
+  for test_id, params in tests_config.items():
```

Comment:
We had spoken about supporting multiple `metric_name` for a single `test_id`. These metric names will be provided as a list in the config. 
Shouldn't we check if `params['metric_name']` is a list and execute `run_change_point_analysis` for each metric_name?

Reply:
This can be done by adding `metric_name` as a parameter to `run_change_point_analysis` instead of accessing it directly in the method?

I'll defer to your opinion